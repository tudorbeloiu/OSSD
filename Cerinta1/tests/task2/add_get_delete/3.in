40
1
6
67
4082
170
3952
219
5048
112
6283
16
8841
116
8609
3
219
1
8
51
2026
115
1770
99
4545
104
152
225
2135
101
3313
220
1074
144
5119
2
16
3
16
1
10
162
802
28
5197
178
2852
141
3928
35
3616
41
3409
20
2409
138
2176
176
7018
222
7030
2
176
1
6
166
2955
92
3994
252
4714
142
4903
83
3465
218
61
3
92
3
28
1
3
191
5422
43
6813
161
4825
1
9
37
2847
165
8987
59
8213
94
4422
86
6104
74
3580
114
1058
93
8987
193
7600
3
218
1
1
196
2641
2
191
3
252
1
1
189
866
1
10
82
6908
173
8687
5
7475
56
2203
198
3744
152
2948
117
7536
39
2555
42
2783
252
7276
2
225
1
9
19
1182
194
8027
26
3291
10
6528
16
3215
246
1805
197
6361
15
8494
48
3887
3
94
3
59
2
99
1
9
68
2543
73
5255
227
4138
188
7705
143
5812
8
3690
146
1847
167
5072
102
2000
1
5
190
984
62
2936
159
8290
23
3378
163
3385
2
26
1
1
236
3310
1
8
63
6304
6
44
231
449
221
4205
235
5599
201
4689
135
5251
127
1971
2
146
1
7
75
5494
140
1639
22
3072
125
2147
128
3136
149
1284
233
4053
3
127
1
3
209
3600
195
7788
207
8602
3
146
1
4
34
4596
223
2171
77
5495
169
5245
3
62
1
4
50
3220
58
5844
109
7712
155
4373
3
77
1
5
240
6163
84
7405
91
6067
106
151
27
1817
1
10
164
8453
184
397
47
8402
247
4970
215
2273
61
1736
89
5834
187
8271
244
3398
255
5805
2
255
