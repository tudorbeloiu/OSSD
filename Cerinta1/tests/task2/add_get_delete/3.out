67: ((0, 0), (0, 510))
170: ((0, 511), (0, 1004))
219: ((1, 0), (1, 630))
112: ((2, 0), (2, 785))
16: ((0, 0), (0, 0))
116: ((0, 0), (0, 0))
67: ((0, 0), (0, 510))
170: ((0, 511), (0, 1004))
112: ((2, 0), (2, 785))
51: ((1, 0), (1, 253))
115: ((1, 254), (1, 475))
99: ((3, 0), (3, 568))
104: ((0, 1005), (0, 1023))
225: ((1, 476), (1, 742))
101: ((3, 569), (3, 983))
220: ((1, 743), (1, 877))
144: ((4, 0), (4, 639))
((0, 0), (0, 0))
67: ((0, 0), (0, 510))
170: ((0, 511), (0, 1004))
104: ((0, 1005), (0, 1023))
51: ((1, 0), (1, 253))
115: ((1, 254), (1, 475))
225: ((1, 476), (1, 742))
220: ((1, 743), (1, 877))
112: ((2, 0), (2, 785))
99: ((3, 0), (3, 568))
101: ((3, 569), (3, 983))
144: ((4, 0), (4, 639))
162: ((1, 878), (1, 978))
28: ((5, 0), (5, 649))
178: ((4, 640), (4, 996))
141: ((6, 0), (6, 490))
35: ((6, 491), (6, 942))
41: ((7, 0), (7, 426))
20: ((5, 650), (5, 951))
138: ((7, 427), (7, 698))
176: ((8, 0), (8, 877))
222: ((9, 0), (9, 878))
((8, 0), (8, 877))
166: ((10, 0), (10, 369))
92: ((10, 370), (10, 869))
252: ((11, 0), (11, 589))
142: ((12, 0), (12, 612))
83: ((11, 590), (11, 1023))
218: ((1, 979), (1, 986))
67: ((0, 0), (0, 510))
170: ((0, 511), (0, 1004))
104: ((0, 1005), (0, 1023))
51: ((1, 0), (1, 253))
115: ((1, 254), (1, 475))
225: ((1, 476), (1, 742))
220: ((1, 743), (1, 877))
162: ((1, 878), (1, 978))
218: ((1, 979), (1, 986))
112: ((2, 0), (2, 785))
99: ((3, 0), (3, 568))
101: ((3, 569), (3, 983))
144: ((4, 0), (4, 639))
178: ((4, 640), (4, 996))
28: ((5, 0), (5, 649))
20: ((5, 650), (5, 951))
141: ((6, 0), (6, 490))
35: ((6, 491), (6, 942))
41: ((7, 0), (7, 426))
138: ((7, 427), (7, 698))
176: ((8, 0), (8, 877))
222: ((9, 0), (9, 878))
166: ((10, 0), (10, 369))
252: ((11, 0), (11, 589))
83: ((11, 590), (11, 1023))
142: ((12, 0), (12, 612))
67: ((0, 0), (0, 510))
170: ((0, 511), (0, 1004))
104: ((0, 1005), (0, 1023))
51: ((1, 0), (1, 253))
115: ((1, 254), (1, 475))
225: ((1, 476), (1, 742))
220: ((1, 743), (1, 877))
162: ((1, 878), (1, 978))
218: ((1, 979), (1, 986))
112: ((2, 0), (2, 785))
99: ((3, 0), (3, 568))
101: ((3, 569), (3, 983))
144: ((4, 0), (4, 639))
178: ((4, 640), (4, 996))
20: ((5, 650), (5, 951))
141: ((6, 0), (6, 490))
35: ((6, 491), (6, 942))
41: ((7, 0), (7, 426))
138: ((7, 427), (7, 698))
176: ((8, 0), (8, 877))
222: ((9, 0), (9, 878))
166: ((10, 0), (10, 369))
252: ((11, 0), (11, 589))
83: ((11, 590), (11, 1023))
142: ((12, 0), (12, 612))
191: ((13, 0), (13, 677))
43: ((14, 0), (14, 851))
161: ((5, 0), (5, 603))
37: ((10, 370), (10, 725))
165: ((0, 0), (0, 0))
59: ((0, 0), (0, 0))
94: ((15, 0), (15, 552))
86: ((16, 0), (16, 762))
74: ((15, 553), (15, 1000))
114: ((2, 786), (2, 918))
93: ((0, 0), (0, 0))
193: ((17, 0), (17, 949))
67: ((0, 0), (0, 510))
170: ((0, 511), (0, 1004))
104: ((0, 1005), (0, 1023))
51: ((1, 0), (1, 253))
115: ((1, 254), (1, 475))
225: ((1, 476), (1, 742))
220: ((1, 743), (1, 877))
162: ((1, 878), (1, 978))
112: ((2, 0), (2, 785))
114: ((2, 786), (2, 918))
99: ((3, 0), (3, 568))
101: ((3, 569), (3, 983))
144: ((4, 0), (4, 639))
178: ((4, 640), (4, 996))
161: ((5, 0), (5, 603))
20: ((5, 650), (5, 951))
141: ((6, 0), (6, 490))
35: ((6, 491), (6, 942))
41: ((7, 0), (7, 426))
138: ((7, 427), (7, 698))
176: ((8, 0), (8, 877))
222: ((9, 0), (9, 878))
166: ((10, 0), (10, 369))
37: ((10, 370), (10, 725))
252: ((11, 0), (11, 589))
83: ((11, 590), (11, 1023))
142: ((12, 0), (12, 612))
191: ((13, 0), (13, 677))
43: ((14, 0), (14, 851))
94: ((15, 0), (15, 552))
74: ((15, 553), (15, 1000))
86: ((16, 0), (16, 762))
193: ((17, 0), (17, 949))
196: ((12, 613), (12, 943))
((13, 0), (13, 677))
67: ((0, 0), (0, 510))
170: ((0, 511), (0, 1004))
104: ((0, 1005), (0, 1023))
51: ((1, 0), (1, 253))
115: ((1, 254), (1, 475))
225: ((1, 476), (1, 742))
220: ((1, 743), (1, 877))
162: ((1, 878), (1, 978))
112: ((2, 0), (2, 785))
114: ((2, 786), (2, 918))
99: ((3, 0), (3, 568))
101: ((3, 569), (3, 983))
144: ((4, 0), (4, 639))
178: ((4, 640), (4, 996))
161: ((5, 0), (5, 603))
20: ((5, 650), (5, 951))
141: ((6, 0), (6, 490))
35: ((6, 491), (6, 942))
41: ((7, 0), (7, 426))
138: ((7, 427), (7, 698))
176: ((8, 0), (8, 877))
222: ((9, 0), (9, 878))
166: ((10, 0), (10, 369))
37: ((10, 370), (10, 725))
83: ((11, 590), (11, 1023))
142: ((12, 0), (12, 612))
196: ((12, 613), (12, 943))
191: ((13, 0), (13, 677))
43: ((14, 0), (14, 851))
94: ((15, 0), (15, 552))
74: ((15, 553), (15, 1000))
86: ((16, 0), (16, 762))
193: ((17, 0), (17, 949))
189: ((7, 699), (7, 807))
82: ((18, 0), (18, 863))
173: ((0, 0), (0, 0))
5: ((19, 0), (19, 934))
56: ((10, 726), (10, 1001))
198: ((11, 0), (11, 467))
152: ((20, 0), (20, 368))
117: ((21, 0), (21, 941))
39: ((13, 678), (13, 997))
42: ((20, 369), (20, 716))
252: ((22, 0), (22, 909))
((1, 476), (1, 742))
19: ((7, 808), (7, 955))
194: ((23, 0), (23, 1003))
26: ((24, 0), (24, 411))
10: ((25, 0), (25, 815))
16: ((24, 412), (24, 813))
246: ((16, 763), (16, 988))
197: ((26, 0), (26, 795))
15: ((0, 0), (0, 0))
48: ((27, 0), (27, 485))
67: ((0, 0), (0, 510))
170: ((0, 511), (0, 1004))
104: ((0, 1005), (0, 1023))
51: ((1, 0), (1, 253))
115: ((1, 254), (1, 475))
225: ((1, 476), (1, 742))
220: ((1, 743), (1, 877))
162: ((1, 878), (1, 978))
112: ((2, 0), (2, 785))
114: ((2, 786), (2, 918))
99: ((3, 0), (3, 568))
101: ((3, 569), (3, 983))
144: ((4, 0), (4, 639))
178: ((4, 640), (4, 996))
161: ((5, 0), (5, 603))
20: ((5, 650), (5, 951))
141: ((6, 0), (6, 490))
35: ((6, 491), (6, 942))
41: ((7, 0), (7, 426))
138: ((7, 427), (7, 698))
189: ((7, 699), (7, 807))
19: ((7, 808), (7, 955))
176: ((8, 0), (8, 877))
222: ((9, 0), (9, 878))
166: ((10, 0), (10, 369))
37: ((10, 370), (10, 725))
56: ((10, 726), (10, 1001))
198: ((11, 0), (11, 467))
83: ((11, 590), (11, 1023))
142: ((12, 0), (12, 612))
196: ((12, 613), (12, 943))
191: ((13, 0), (13, 677))
39: ((13, 678), (13, 997))
43: ((14, 0), (14, 851))
74: ((15, 553), (15, 1000))
86: ((16, 0), (16, 762))
246: ((16, 763), (16, 988))
193: ((17, 0), (17, 949))
82: ((18, 0), (18, 863))
5: ((19, 0), (19, 934))
152: ((20, 0), (20, 368))
42: ((20, 369), (20, 716))
117: ((21, 0), (21, 941))
252: ((22, 0), (22, 909))
194: ((23, 0), (23, 1003))
26: ((24, 0), (24, 411))
16: ((24, 412), (24, 813))
10: ((25, 0), (25, 815))
197: ((26, 0), (26, 795))
48: ((27, 0), (27, 485))
67: ((0, 0), (0, 510))
170: ((0, 511), (0, 1004))
104: ((0, 1005), (0, 1023))
51: ((1, 0), (1, 253))
115: ((1, 254), (1, 475))
225: ((1, 476), (1, 742))
220: ((1, 743), (1, 877))
162: ((1, 878), (1, 978))
112: ((2, 0), (2, 785))
114: ((2, 786), (2, 918))
99: ((3, 0), (3, 568))
101: ((3, 569), (3, 983))
144: ((4, 0), (4, 639))
178: ((4, 640), (4, 996))
161: ((5, 0), (5, 603))
20: ((5, 650), (5, 951))
141: ((6, 0), (6, 490))
35: ((6, 491), (6, 942))
41: ((7, 0), (7, 426))
138: ((7, 427), (7, 698))
189: ((7, 699), (7, 807))
19: ((7, 808), (7, 955))
176: ((8, 0), (8, 877))
222: ((9, 0), (9, 878))
166: ((10, 0), (10, 369))
37: ((10, 370), (10, 725))
56: ((10, 726), (10, 1001))
198: ((11, 0), (11, 467))
83: ((11, 590), (11, 1023))
142: ((12, 0), (12, 612))
196: ((12, 613), (12, 943))
191: ((13, 0), (13, 677))
39: ((13, 678), (13, 997))
43: ((14, 0), (14, 851))
74: ((15, 553), (15, 1000))
86: ((16, 0), (16, 762))
246: ((16, 763), (16, 988))
193: ((17, 0), (17, 949))
82: ((18, 0), (18, 863))
5: ((19, 0), (19, 934))
152: ((20, 0), (20, 368))
42: ((20, 369), (20, 716))
117: ((21, 0), (21, 941))
252: ((22, 0), (22, 909))
194: ((23, 0), (23, 1003))
26: ((24, 0), (24, 411))
16: ((24, 412), (24, 813))
10: ((25, 0), (25, 815))
197: ((26, 0), (26, 795))
48: ((27, 0), (27, 485))
((3, 0), (3, 568))
68: ((15, 0), (15, 317))
73: ((28, 0), (28, 656))
227: ((27, 486), (27, 1003))
188: ((29, 0), (29, 963))
143: ((30, 0), (30, 726))
8: ((31, 0), (31, 461))
146: ((15, 318), (15, 548))
167: ((32, 0), (32, 633))
102: ((20, 717), (20, 966))
190: ((8, 878), (8, 1000))
62: ((28, 657), (28, 1023))
159: ((0, 0), (0, 0))
23: ((31, 462), (31, 884))
163: ((33, 0), (33, 423))
((24, 0), (24, 411))
236: ((33, 424), (33, 837))
63: ((34, 0), (34, 787))
6: ((1, 979), (1, 984))
231: ((2, 919), (2, 975))
221: ((35, 0), (35, 525))
235: ((36, 0), (36, 699))
201: ((37, 0), (37, 586))
135: ((38, 0), (38, 656))
127: ((30, 727), (30, 973))
((15, 318), (15, 548))
75: ((39, 0), (39, 686))
140: ((24, 814), (24, 1018))
22: ((32, 634), (32, 1017))
125: ((35, 526), (35, 794))
128: ((37, 587), (37, 978))
149: ((14, 852), (14, 1012))
233: ((40, 0), (40, 506))
67: ((0, 0), (0, 510))
170: ((0, 511), (0, 1004))
104: ((0, 1005), (0, 1023))
51: ((1, 0), (1, 253))
115: ((1, 254), (1, 475))
225: ((1, 476), (1, 742))
220: ((1, 743), (1, 877))
162: ((1, 878), (1, 978))
6: ((1, 979), (1, 984))
112: ((2, 0), (2, 785))
114: ((2, 786), (2, 918))
231: ((2, 919), (2, 975))
99: ((3, 0), (3, 568))
101: ((3, 569), (3, 983))
144: ((4, 0), (4, 639))
178: ((4, 640), (4, 996))
161: ((5, 0), (5, 603))
20: ((5, 650), (5, 951))
141: ((6, 0), (6, 490))
35: ((6, 491), (6, 942))
41: ((7, 0), (7, 426))
138: ((7, 427), (7, 698))
189: ((7, 699), (7, 807))
19: ((7, 808), (7, 955))
176: ((8, 0), (8, 877))
190: ((8, 878), (8, 1000))
222: ((9, 0), (9, 878))
166: ((10, 0), (10, 369))
37: ((10, 370), (10, 725))
56: ((10, 726), (10, 1001))
198: ((11, 0), (11, 467))
83: ((11, 590), (11, 1023))
142: ((12, 0), (12, 612))
196: ((12, 613), (12, 943))
191: ((13, 0), (13, 677))
39: ((13, 678), (13, 997))
43: ((14, 0), (14, 851))
149: ((14, 852), (14, 1012))
68: ((15, 0), (15, 317))
146: ((15, 318), (15, 548))
74: ((15, 553), (15, 1000))
86: ((16, 0), (16, 762))
246: ((16, 763), (16, 988))
193: ((17, 0), (17, 949))
82: ((18, 0), (18, 863))
5: ((19, 0), (19, 934))
152: ((20, 0), (20, 368))
42: ((20, 369), (20, 716))
102: ((20, 717), (20, 966))
117: ((21, 0), (21, 941))
252: ((22, 0), (22, 909))
194: ((23, 0), (23, 1003))
26: ((24, 0), (24, 411))
16: ((24, 412), (24, 813))
140: ((24, 814), (24, 1018))
10: ((25, 0), (25, 815))
197: ((26, 0), (26, 795))
48: ((27, 0), (27, 485))
227: ((27, 486), (27, 1003))
73: ((28, 0), (28, 656))
62: ((28, 657), (28, 1023))
188: ((29, 0), (29, 963))
143: ((30, 0), (30, 726))
8: ((31, 0), (31, 461))
23: ((31, 462), (31, 884))
167: ((32, 0), (32, 633))
22: ((32, 634), (32, 1017))
163: ((33, 0), (33, 423))
236: ((33, 424), (33, 837))
63: ((34, 0), (34, 787))
221: ((35, 0), (35, 525))
125: ((35, 526), (35, 794))
235: ((36, 0), (36, 699))
201: ((37, 0), (37, 586))
128: ((37, 587), (37, 978))
135: ((38, 0), (38, 656))
75: ((39, 0), (39, 686))
233: ((40, 0), (40, 506))
209: ((40, 507), (40, 956))
195: ((41, 0), (41, 973))
207: ((0, 0), (0, 0))
67: ((0, 0), (0, 510))
170: ((0, 511), (0, 1004))
104: ((0, 1005), (0, 1023))
51: ((1, 0), (1, 253))
115: ((1, 254), (1, 475))
225: ((1, 476), (1, 742))
220: ((1, 743), (1, 877))
162: ((1, 878), (1, 978))
6: ((1, 979), (1, 984))
112: ((2, 0), (2, 785))
114: ((2, 786), (2, 918))
231: ((2, 919), (2, 975))
99: ((3, 0), (3, 568))
101: ((3, 569), (3, 983))
144: ((4, 0), (4, 639))
178: ((4, 640), (4, 996))
161: ((5, 0), (5, 603))
20: ((5, 650), (5, 951))
141: ((6, 0), (6, 490))
35: ((6, 491), (6, 942))
41: ((7, 0), (7, 426))
138: ((7, 427), (7, 698))
189: ((7, 699), (7, 807))
19: ((7, 808), (7, 955))
176: ((8, 0), (8, 877))
190: ((8, 878), (8, 1000))
222: ((9, 0), (9, 878))
166: ((10, 0), (10, 369))
37: ((10, 370), (10, 725))
56: ((10, 726), (10, 1001))
198: ((11, 0), (11, 467))
83: ((11, 590), (11, 1023))
142: ((12, 0), (12, 612))
196: ((12, 613), (12, 943))
191: ((13, 0), (13, 677))
39: ((13, 678), (13, 997))
43: ((14, 0), (14, 851))
149: ((14, 852), (14, 1012))
68: ((15, 0), (15, 317))
74: ((15, 553), (15, 1000))
86: ((16, 0), (16, 762))
246: ((16, 763), (16, 988))
193: ((17, 0), (17, 949))
82: ((18, 0), (18, 863))
5: ((19, 0), (19, 934))
152: ((20, 0), (20, 368))
42: ((20, 369), (20, 716))
102: ((20, 717), (20, 966))
117: ((21, 0), (21, 941))
252: ((22, 0), (22, 909))
194: ((23, 0), (23, 1003))
26: ((24, 0), (24, 411))
16: ((24, 412), (24, 813))
140: ((24, 814), (24, 1018))
10: ((25, 0), (25, 815))
197: ((26, 0), (26, 795))
48: ((27, 0), (27, 485))
227: ((27, 486), (27, 1003))
73: ((28, 0), (28, 656))
62: ((28, 657), (28, 1023))
188: ((29, 0), (29, 963))
143: ((30, 0), (30, 726))
8: ((31, 0), (31, 461))
23: ((31, 462), (31, 884))
167: ((32, 0), (32, 633))
22: ((32, 634), (32, 1017))
163: ((33, 0), (33, 423))
236: ((33, 424), (33, 837))
63: ((34, 0), (34, 787))
221: ((35, 0), (35, 525))
125: ((35, 526), (35, 794))
235: ((36, 0), (36, 699))
201: ((37, 0), (37, 586))
128: ((37, 587), (37, 978))
135: ((38, 0), (38, 656))
75: ((39, 0), (39, 686))
233: ((40, 0), (40, 506))
209: ((40, 507), (40, 956))
195: ((41, 0), (41, 973))
34: ((42, 0), (42, 574))
223: ((30, 727), (30, 998))
77: ((43, 0), (43, 686))
169: ((44, 0), (44, 655))
67: ((0, 0), (0, 510))
170: ((0, 511), (0, 1004))
104: ((0, 1005), (0, 1023))
51: ((1, 0), (1, 253))
115: ((1, 254), (1, 475))
225: ((1, 476), (1, 742))
220: ((1, 743), (1, 877))
162: ((1, 878), (1, 978))
6: ((1, 979), (1, 984))
112: ((2, 0), (2, 785))
114: ((2, 786), (2, 918))
231: ((2, 919), (2, 975))
99: ((3, 0), (3, 568))
101: ((3, 569), (3, 983))
144: ((4, 0), (4, 639))
178: ((4, 640), (4, 996))
161: ((5, 0), (5, 603))
20: ((5, 650), (5, 951))
141: ((6, 0), (6, 490))
35: ((6, 491), (6, 942))
41: ((7, 0), (7, 426))
138: ((7, 427), (7, 698))
189: ((7, 699), (7, 807))
19: ((7, 808), (7, 955))
176: ((8, 0), (8, 877))
190: ((8, 878), (8, 1000))
222: ((9, 0), (9, 878))
166: ((10, 0), (10, 369))
37: ((10, 370), (10, 725))
56: ((10, 726), (10, 1001))
198: ((11, 0), (11, 467))
83: ((11, 590), (11, 1023))
142: ((12, 0), (12, 612))
196: ((12, 613), (12, 943))
191: ((13, 0), (13, 677))
39: ((13, 678), (13, 997))
43: ((14, 0), (14, 851))
149: ((14, 852), (14, 1012))
68: ((15, 0), (15, 317))
74: ((15, 553), (15, 1000))
86: ((16, 0), (16, 762))
246: ((16, 763), (16, 988))
193: ((17, 0), (17, 949))
82: ((18, 0), (18, 863))
5: ((19, 0), (19, 934))
152: ((20, 0), (20, 368))
42: ((20, 369), (20, 716))
102: ((20, 717), (20, 966))
117: ((21, 0), (21, 941))
252: ((22, 0), (22, 909))
194: ((23, 0), (23, 1003))
26: ((24, 0), (24, 411))
16: ((24, 412), (24, 813))
140: ((24, 814), (24, 1018))
10: ((25, 0), (25, 815))
197: ((26, 0), (26, 795))
48: ((27, 0), (27, 485))
227: ((27, 486), (27, 1003))
73: ((28, 0), (28, 656))
188: ((29, 0), (29, 963))
143: ((30, 0), (30, 726))
223: ((30, 727), (30, 998))
8: ((31, 0), (31, 461))
23: ((31, 462), (31, 884))
167: ((32, 0), (32, 633))
22: ((32, 634), (32, 1017))
163: ((33, 0), (33, 423))
236: ((33, 424), (33, 837))
63: ((34, 0), (34, 787))
221: ((35, 0), (35, 525))
125: ((35, 526), (35, 794))
235: ((36, 0), (36, 699))
201: ((37, 0), (37, 586))
128: ((37, 587), (37, 978))
135: ((38, 0), (38, 656))
75: ((39, 0), (39, 686))
233: ((40, 0), (40, 506))
209: ((40, 507), (40, 956))
195: ((41, 0), (41, 973))
34: ((42, 0), (42, 574))
77: ((43, 0), (43, 686))
169: ((44, 0), (44, 655))
50: ((42, 575), (42, 977))
58: ((45, 0), (45, 730))
109: ((46, 0), (46, 963))
155: ((47, 0), (47, 546))
67: ((0, 0), (0, 510))
170: ((0, 511), (0, 1004))
104: ((0, 1005), (0, 1023))
51: ((1, 0), (1, 253))
115: ((1, 254), (1, 475))
225: ((1, 476), (1, 742))
220: ((1, 743), (1, 877))
162: ((1, 878), (1, 978))
6: ((1, 979), (1, 984))
112: ((2, 0), (2, 785))
114: ((2, 786), (2, 918))
231: ((2, 919), (2, 975))
99: ((3, 0), (3, 568))
101: ((3, 569), (3, 983))
144: ((4, 0), (4, 639))
178: ((4, 640), (4, 996))
161: ((5, 0), (5, 603))
20: ((5, 650), (5, 951))
141: ((6, 0), (6, 490))
35: ((6, 491), (6, 942))
41: ((7, 0), (7, 426))
138: ((7, 427), (7, 698))
189: ((7, 699), (7, 807))
19: ((7, 808), (7, 955))
176: ((8, 0), (8, 877))
190: ((8, 878), (8, 1000))
222: ((9, 0), (9, 878))
166: ((10, 0), (10, 369))
37: ((10, 370), (10, 725))
56: ((10, 726), (10, 1001))
198: ((11, 0), (11, 467))
83: ((11, 590), (11, 1023))
142: ((12, 0), (12, 612))
196: ((12, 613), (12, 943))
191: ((13, 0), (13, 677))
39: ((13, 678), (13, 997))
43: ((14, 0), (14, 851))
149: ((14, 852), (14, 1012))
68: ((15, 0), (15, 317))
74: ((15, 553), (15, 1000))
86: ((16, 0), (16, 762))
246: ((16, 763), (16, 988))
193: ((17, 0), (17, 949))
82: ((18, 0), (18, 863))
5: ((19, 0), (19, 934))
152: ((20, 0), (20, 368))
42: ((20, 369), (20, 716))
102: ((20, 717), (20, 966))
117: ((21, 0), (21, 941))
252: ((22, 0), (22, 909))
194: ((23, 0), (23, 1003))
26: ((24, 0), (24, 411))
16: ((24, 412), (24, 813))
140: ((24, 814), (24, 1018))
10: ((25, 0), (25, 815))
197: ((26, 0), (26, 795))
48: ((27, 0), (27, 485))
227: ((27, 486), (27, 1003))
73: ((28, 0), (28, 656))
188: ((29, 0), (29, 963))
143: ((30, 0), (30, 726))
223: ((30, 727), (30, 998))
8: ((31, 0), (31, 461))
23: ((31, 462), (31, 884))
167: ((32, 0), (32, 633))
22: ((32, 634), (32, 1017))
163: ((33, 0), (33, 423))
236: ((33, 424), (33, 837))
63: ((34, 0), (34, 787))
221: ((35, 0), (35, 525))
125: ((35, 526), (35, 794))
235: ((36, 0), (36, 699))
201: ((37, 0), (37, 586))
128: ((37, 587), (37, 978))
135: ((38, 0), (38, 656))
75: ((39, 0), (39, 686))
233: ((40, 0), (40, 506))
209: ((40, 507), (40, 956))
195: ((41, 0), (41, 973))
34: ((42, 0), (42, 574))
50: ((42, 575), (42, 977))
169: ((44, 0), (44, 655))
58: ((45, 0), (45, 730))
109: ((46, 0), (46, 963))
155: ((47, 0), (47, 546))
240: ((43, 0), (43, 770))
84: ((48, 0), (48, 925))
91: ((49, 0), (49, 758))
106: ((1, 985), (1, 1003))
27: ((15, 318), (15, 545))
164: ((0, 0), (0, 0))
184: ((5, 952), (5, 1001))
47: ((0, 0), (0, 0))
247: ((50, 0), (50, 621))
215: ((28, 657), (28, 941))
61: ((26, 796), (26, 1012))
89: ((51, 0), (51, 729))
187: ((0, 0), (0, 0))
244: ((47, 547), (47, 971))
255: ((52, 0), (52, 725))
((52, 0), (52, 725))
